frontend/components/TabEach.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const TabEachComponent = ({ currentPage, info }) => {
```

Comment:
TabEach 컴포넌트의 이름으로 TabButton은 어떠신가요?
```suggestion
const TabButton = ({ currentPage, info }) => {
```